header/header_modifier.go

```diff
@@ -74,5 +95,13 @@ func modifierFromJSON(b []byte) (*parse.Result, error) {
 
 	modifier := NewModifier(msg.Name, msg.Value)
 
+	if msg.Behavior == "" || msg.Behavior == "replace" {
```

Comment:
if the default behavior is set in NewModifier, you should only need to do:
```
if strings.ToLower(msg.Behavior) == "append" {
  modifier.SetBehavior(AppendValues)
}
```

Reply:
You could just assume that anything that isn't `append` gives you the default `replace` behavior.

But, if you'd like to be explicit and return an error on a value that is neither append or default, use a switch:

```
switch strings.ToLower(msg.Behavior) {
case "append":
  modifier.SetBehavior(AppendValues)
case "replace":
  modifier.SetBehavior(ReplaceValues)
default:
  return nil, fmt.Errorf("Invalid header modifier behavior %q. Must be either %q or %q", msg.Behavior, "append", "replace")
}
```

Reply:
Went with a switch here, I think silently ignoring the other val would be annoying to debug